### ISSUE TITLE
"Hide the tab bar" does not work.

### ISSUE THREAD

**Issue by MememeOyaji** [onemen/TabMixPlus#336]
Firefox 128.3.1esr 64bit,  Windows10 22H1,  Tab Mix Plus v1.24.0
Display > Tab Bar > Hide the tab bar
Tab bar can not be hidden when I choose "When you have only one tab".
When I downgrade TMP to v1.23.0, it works normally.

**Comment by onemen** [comment]
I can not reproduce this issue

- Attach a file with your Tab mix preferences 
- Do you see any message in the browser console when you close 2nd tab, leaving you with one tab

**Comment by MememeOyaji** [comment]
The Development Build works good.
That bug does not appear with this build.
Thanks.

**Comment by MememeOyaji** [comment]
Now I made a new profile, installed only TMP (no other addons).
The only one entry I changed is "Hide the tab bar".

V1.24.0 still showed the bug.
Uninstalled v1.24.0, cleared startup chache, installed dev-build, the bug disappeared.
Then I installed again v1.24.0 without uninstalling dev-build, without clearing startup cache,
the bug did not appear.
The bug appeared again when I cleared start up cache.
